cmd/integration_test/build/buildspecs/test-eks-a-cli.yml

```diff
@@ -5,7 +5,7 @@ env:
     INTEGRATION_TEST_MAX_EC2_COUNT: 165
     INTEGRATION_TEST_MAX_CONCURRENT_TEST_COUNT: 165
     T_CLOUDSTACK_CIDR: "10.11.255.0/24"
-    SKIPPED_TESTS: "TestTinkerbellKubernetes120AWSIamAuth,TestTinkerbellKubernetes120SimpleFlow,TestTinkerbellKubernetes121ExternalEtcdSimpleFlow,TestVSphereKubernetes122BottlerocketAutoimport,TestCloudStackKubernetes120OIDC,TestCloudStackKubernetes121OIDC,TestCloudStackKubernetes121RedhatRegistryMirrorAndCert,TestSnowKubernetes121SimpleFlow,TestSnowKubernetes122SimpleFlow,TestSnowKubernetes123SimpleFlow,TestSnowKubernetes121OIDC,TestSnowKubernetes121UbuntuProxyConfig,TestSnowKubernetes123UbuntuRemoveWorkerNodeGroups,TestSnowKubernetes123LabelsUbuntu,TestSnowKubernetes123TaintsUbuntu,TestSnowKubernetes121To122UbuntuUpgrade,TestSnowKubernetes122To123UbuntuMultipleFieldsUpgrade"
+    SKIPPED_TESTS: "TestTinkerbellKubernetes122SkipPowerActions,TestTinkerbellKubernetes120AWSIamAuth,TestTinkerbellKubernetes120SimpleFlow,TestTinkerbellKubernetes121ExternalEtcdSimpleFlow,TestVSphereKubernetes122BottlerocketAutoimport,TestCloudStackKubernetes120OIDC,TestCloudStackKubernetes121OIDC,TestCloudStackKubernetes121RedhatRegistryMirrorAndCert,TestSnowKubernetes121SimpleFlow,TestSnowKubernetes122SimpleFlow,TestSnowKubernetes123SimpleFlow,TestSnowKubernetes121OIDC,TestSnowKubernetes121UbuntuProxyConfig,TestSnowKubernetes123UbuntuRemoveWorkerNodeGroups,TestSnowKubernetes123LabelsUbuntu,TestSnowKubernetes123TaintsUbuntu,TestSnowKubernetes121To122UbuntuUpgrade,TestSnowKubernetes122To123UbuntuMultipleFieldsUpgrade"
```

Comment:
Is this related to this PR?

Reply:
Sort of.. This PR started as an effort to add more resiliency to the tinkerbell e2e tests which ultimately led to wanting ability to override timeouts so when tink test fails we don't have to wait whole 60m. 

This is a test that is flaky and I don't have a great option to fix it since to do so I would need to reformat the disk on each test run. Therefore, I decided to skip it for now so it stops polluting the build with failures until I can come up with a way to resolve the intermittency associated with it.

Reply:
I can submit in different PR if you prefer.

Reply:
Non blocking, Up to you :)
Different PRs is always better, but I'm ok with this, as long there was a reason for it and there is